Isu/Models/StudentGroup.cs

```diff
@@ -0,0 +1,84 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.RegularExpressions;
+using Isu.Services;
+using Isu.Tools;
+
+namespace Isu.Models
+{
+    public class StudentGroup
+    {
+        public List<Group> Group { get; } = new List<Group>();
+        public List<Student> Student { get; } = new List<Student>();
+
+        public Group AddGroup(string name)
+        {
+            Group.Add(new Group() { GroupName = name });
+            return Group[^1];
```

Comment:
Вместо срезов лучше использовать
```suggestion
            return Group.Last();
```

А так вообще лучше вынести создание группы в отдельную